scripts/local/docker-compose.yml

```diff
@@ -0,0 +1,146 @@
+# This is the docker-compose script for local development.
+# For the latest version of the docker images see:
+# https://github.com/Azure/pcs-cli/blob/master/remotemonitoring/single-vm/docker-compose.dotnet.yml
+#
+# Usage:
+# - install Docker Compose, see https://docs.docker.com/compose/install
+# - open a console and `cd` into the folder that contains this file
+# - note how the file references many environment variables, either set
+#   them in the .env file or edit this file adding the values here
+# - run `docker-compose up`
+# - open the browser at http://127.0.0.1:10443
+#
+# For more information see
+# https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet
+
+version: "3"
+
+services:
+  reverseproxy:
+    image: azureiotpcs/remote-monitoring-nginx:1.0.0-preview.3
```

Comment:
I think we could use the "testing" tag, wdyt?

Reply:
Testing should be fine, if you make it to the local development steps, the latest in our repos could be best. The only concern was if someone just wants to make a single change to one web-service and test against that.